github_stargazers/github.py

```diff
@@ -17,6 +17,12 @@ def __init__(self) -> None:
         super().__init__("Too many requests.")
 
 
+class UrlNotFound(Exception):
```

Comment:
Let's keep consistency with the rest of defined Exceptions and rename it to `UrlNotFoundError`.